tests/fixtures_data_analyzer.py

```diff
@@ -9,6 +11,15 @@
 from .fixtures import QUEUE_CHECK_TIMEOUT_SECONDS
 from .helpers import put_object_into_queue_and_raise_error_if_eventually_still_empty
 
+TEST_RECORDING_ROOT_PATH = os.path.join(os.path.abspath("tests"), "test_recording_h5")
+TEST_REC_DIR_PATH = os.path.join(TEST_RECORDING_ROOT_PATH, "ML2021172153__2022_01_21_023323")
```

Comment:
Could probably use `tempfile` to create a `TemporaryFile` wherever a real file is needed instead of committing an empty file to the repo just for rest purposes